EssentialsChat/src/main/java/com/earth2me/essentials/chat/EssentialsChat.java

```diff
@@ -47,7 +47,6 @@ public void onEnable() {
 
     @Override
     public boolean onCommand(final CommandSender sender, final Command command, final String commandLabel, final String[] args) {
-        metrics.markCommand(command.getName(), true);
```

Comment:
oof

Reply:
intended

Reply:
> Where is this replaced?

don't remember but I also didn't to this in the discord module also i don't know why we even do this when we don't use them in the first place

Reply:
This is for future metrics stuff, I'm waiting on either bStats to support the right graph or for us to adopt another metrics solution.